docs/_doc/Creating-a-local-cluster.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Creating a local cluster
+---
+
+<!-- TODO: Redirects -->
+
+> In the first link, the first section "Setting up multiple nodes" does not necessarily imply that those nodes will be connected in any way. They could literally be separate nodes that have no knowledge of each other. Perhaps serving as an array of nodes for some local testers, for example.
```

Comment:
I am confused by these comments, doesn't `admin.addPeer(enodeUrlOfFirstInstance)` mean they are aware of each other? Or did I end up removing the content that makes them unaware?

Reply:
Yeah, that should indeed make them aware of eachother, imo